{
	"header": {
		"line": "CVE-2024-7885 (A vulnerability was found in Undertow where the ProxyProtocolReadListe ...)",
		"name": "CVE-2024-7885",
		"description": "(A vulnerability was found in Undertow where the ProxyProtocolReadListe ...)"
	},
	"annotations": [
		{
			"line": "\t- undertow <unfixed> (bug #1082854)",
			"type": "package",
			"package": "undertow",
			"kind": "unfixed",
			"flags": [
				{
					"bug": 1082854
				}
			]
		},
		{
			"line": "\tNOTE: https://bugzilla.redhat.com/show_bug.cgi?id=2305290",
			"type": "NOTE",
			"description": "https://bugzilla.redhat.com/show_bug.cgi?id=2305290"
		},
		{
			"line": "\tNOTE: Fixed by: https://github.com/undertow-io/undertow/commit/ce5182c37376982ef0abee34fce0d8c0aab0fab8 (2.2.36.Final)",
			"type": "NOTE",
			"description": "Fixed by: https://github.com/undertow-io/undertow/commit/ce5182c37376982ef0abee34fce0d8c0aab0fab8 (2.2.36.Final)"
		},
		{
			"line": "\tNOTE: Fixed by: https://github.com/undertow-io/undertow/commit/80c125e09068ac52ed0a9acde266ef12f8ed7ae1 (2.3.17.Final)",
			"type": "NOTE",
			"description": "Fixed by: https://github.com/undertow-io/undertow/commit/80c125e09068ac52ed0a9acde266ef12f8ed7ae1 (2.3.17.Final)"
		}
	]
}
